src/amo/actions/featured.js

```diff
@@ -0,0 +1,31 @@
+import {
+  FEATURED_GET,
+  FEATURED_LOADED,
+  FEATURED_FAILED,
+} from 'core/constants';
+
+
+export function getFeatured({ addonType }) {
+  if (!addonType) {
+    throw new Error('addonType must be set');
+  }
+
+  return {
+    type: FEATURED_GET,
+    payload: { addonType },
+  };
+}
+
+export function loadFeatured({ addonType, entities, result }) {
+  return {
+    type: FEATURED_LOADED,
+    payload: { addonType, entities, result },
+  };
+}
+
+export function failFeatured({ addonType }) {
```

Comment:
Fair enough. It at least shows a failure action during development but I guess that's it. I'll remove them, but in their own commit so we can always easily fetch them if wanted. 😄